VERSION

```diff
@@ -1,3 +1,7 @@
+#ifdef OMPI_BUILDING_CXX_BINDINGS_LIBRARY
```

Comment:
Please don't do this.  I can't understand how stdc++ did something so stupid, but adding C code in a contextual text file is a terrible idea.  Let's try again; perhaps the right answer is figuring out how to remove the -I to the top level of the project.  But this is  just asking for trouble.